drivers/sensor/ina260/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+#
+#  Copyright (c) 2024 Richie Sommers
```

Comment:
Just change all copyrights to RIT Launch Initiative in this repo moving forward. If you contribute this to Zephyr, then you can put your own name.